tests/recipes/cec1/baseline/test_evaluate.py

```diff
@@ -31,15 +31,21 @@ def not_tqdm(iterable):
 def truncated_read_signal(
```

Comment:
Would it be reasonable to make `utils.file_io.read_signal()` have an option of how much of a signal to return (or write for `utils.file_io.write_signal()`)?

Reply:
Thanks. Actually, read_signal already has that. So this should be rewritten to use it, i.e like

```
signal = read_signal(
        filename,
        offset=offset,
        n_samples=n_samples,
        n_channels=n_channels,
        offset_is_samples=offset_is_samples,
        n_samples=44100,
    )
```